Registry/Registry.EM_COMMON

```diff
@@ -2267,6 +2272,8 @@ rconfig   integer     do_radar_ref        namelist,physics      1              0
 rconfig   integer     compute_radar_ref   derived               1              0       -        "compute_radar_ref" "0/1 flag: compute radar reflectivity, either do_radar_ref=1 .or. (milbrandt or NSSL schemes)"
 rconfig   integer     ra_lw_physics       namelist,physics	max_domains    -1      rh       "ra_lw_physics"         ""      ""
 rconfig   integer     ra_sw_physics       namelist,physics	max_domains    -1      rh       "ra_sw_physics"         ""      ""
+amontornes-bcodina 2015/09
+rconfig   integer     ra_sw_eclipse       namelist,physics	1              0       rh       "ra_sw_eclipse"         ""      ""
```

Comment:
@irowebbn 
Isaac,
1. Remove the developer citation line
2. If this is a 0/1 flag, mention that - similar as is done with the compute_radar_ref a few lines up
3. If this is the option that activates the eclipse computations, this is the variable that is to be used in the packaging

Reply:
It appears that the namelist, ra_sw_eclipse, is not used in the code.

Reply:
Addressed partially.

Reply:
All addressed.